Makefile

```diff
@@ -22,10 +22,6 @@ PREFIX		?= arm-none-eabi-
 STYLECHECK      := scripts/checkpatch.pl
 STYLECHECKFLAGS := --no-tree -f --terse --mailback
 
-space:=
-space+=
-SRCLIBDIR:= $(subst $(space),\$(space),$(realpath lib))
```

Comment:
where does this get set now?

Reply:
This has been set in the target Makefiles for a while already, so for example lib/stm32/f0/Makefile.